spec/ttml3.xml

```diff
@@ -1147,6 +1147,16 @@ attribute of a <loc href="#embedded-content-vocabulary-source"><el>source</el></
 content element</loc>.</p>
 </def>
 </gitem>
+<gitem id="terms-external-module">
+<label>[external module]</label>
+<def>
+<p>A module defined externally to this specification in a separate module defining document. An external module
+is designated at the time of specification as either a W3C defined module, in which case any vocabulary or semantic
+behavior it defines is specified in the scope of one or more TTML Namespaces listed in <specref ref="namespace-vocab-table"/>,
```

Comment:
The idea of supporting 3rd party modules is appealing, but needs an additional term such as `external specification module` or `private specification module`. I would rather not mix in namespace constraints with the term definition for modules; that presupposes decisions we have not yet made. For example we may choose to define a namespace explicitly for use in 3rd party extension modules for some purpose.

Reply:
I disagree with your premise that would have module be equated to a specification. That is not the case, or at least not the case with the pre-existing (internal) modules. What we are defining now is a way to create external modules, whether public (listed in registry) or private (not listed in registry).

Reply:
I agree that a module is not a specification, and am happy to think about other terms.

Reply:
Try "specification collection".

Reply:
I'm really confused at this point about whether you consider a module to be a specification or not @skynavga .